### PR TITLE
Newdesign

### PR DESCRIPTION
**Pull request by Techylite**
we have deleted some files and corrected some code

### CODE REVIEW

File: spec/controllers/Admin/profile_controller_spec.rb [hunk 1]
```diff
@@ -1,21 +1,281 @@
+require 'spec_helper'
```
Comment (tyranja) [inline spec/controllers/Admin/profile_controller_spec.rb]: Isn't that the 'spec_helper' already required in the .rspec file? So we don't need that here?

**Comment by tyranja** [comment]
Cool! :yum:

**Comment by zaziemo** [comment]
Good work @Techylite :) Would you please have a look on @tyranja 's question? Than I can merge your Pull Request!

**Comment by Techylite** [comment]
oh yah @tyranja lets get rid of it

**Pull request merged**